File: 724. Find Pivot Index/solution.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        sum_right = 0
        for x in nums:
            sum_right += x

        left = 0
        for x in range(len(nums)):
            if(left == sum_right - nums[x] - left):
                return x
            left += nums[x]
        return -1

File: 724. Find Pivot Index/solution.java
class Solution {
    public int pivotIndex(int[] nums) {
        var sum_arr = 0;
        var left = 0;
        for (int x : nums)
            sum_arr += x;
        for (int i = 0; i < nums.length; i++) {
            if (left == sum_arr - left - nums[i])
                return i;
            left += nums[i];
        }
        return -1;
    }
}